disdrodb/utils/zenodo.py

```diff
@@ -0,0 +1,74 @@
+import os
+import requests
+from typing import Tuple
+
+
+def _get_zenodo_access_token(sandbox: bool = False) -> str:
+    """Get Zenodo access token from environment variable or from user input."""
+
+    access_token = os.environ.get("ZENODO_ACCESS_TOKEN")
+
+    if not access_token:
```

Comment:
Ok nice! This eventually will be updated if we create the disdrodb config yaml file.